packages/loader/container-loader/src/container.ts

```diff
@@ -1036,6 +1048,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     private async createDetached(source: IFluidCodeDetails) {
+        if (!isFluidCodeDetails(source)) {
+            this.logger.send({
```

Comment:
Not sure what's the intention here, but I'd lean further on fail-fast approach (i.e. at least issue an error event)

Reply:
just trying to get a picture of what is being used, so i can do the right things for forward/back compat